WcaOnRails/app/models/event.rb

```diff
@@ -54,4 +54,20 @@ def fewest_moves?
   def multiple_blindfolded?
     self.id == "333mbf" || self.id == "333mbo"
   end
+
+  def can_change_time_limit?
+    !fewest_moves? && !multiple_blindfolded?
+  end
+
+  def serializable_hash(options = nil)
+    {
+      id: self.id,
+      name: self.name,
+      formatIds: self.formats.map(&:id),
+      canChangeTimeLimit: self.can_change_time_limit?,
+      isTimedEvent: self.timed_event?,
+      isFewestMoves: self.fewest_moves?,
+      isMultipleBlindfolded: self.multiple_blindfolded?,
```

Comment:
Why do you prefer having camelCase on the ruby side instead of transforming the keys as suggested [here](https://github.com/thewca/worldcubeassociation.org/pull/1856/files/6ad50b1c749d4140caca8f94d2781498dd7eea0a..e7e08e45fcd9121a3e8868d24ff0b6e66e2b4410#r137937784)?

Reply:
Ah! I did start out with the camelCase transformation as you suggested, but then I realized for full javascriptiness, it's nicer to have `event.isTimedEvent` rather than `event.timedEvent`. Alternatively, we could do the camelcasing in javascript and live without `is*` prefixes, or we could have some heuristic that prefixes boolean values with "is", unless they already start with "is" or "can". What do you think is best?

Reply:
So I don't see a big problem with using the same name (and other case) in both worlds, so `is_multiple_blinfolded` looks ok as ruby parameter (unless Rubocop things otherwise?), and if `event.timed_event` is meaningful enough in ruby, why wouldn't `event.timedEvent` be in JS?

Reply:
Ah, I like you sugggestion to put the `is_` over in Ruby land. I would argue that `event.timed_event` is not very meaningful in ruby, whereas `event.timed_event?` *is*. Unfortunately, we don't have the suffix with "?" trick in Javascript land.

Reply:
Also, unfortunately, `_.mapKeys` receives it's arguments as `value, key`, so I had to rework things a bit. Please take a look after my next commit and let me know what you think!

Reply:
Ah, that's right, sorry for the invalid line I suggested. Currently it looks great!